lib/utils/logger.js

```diff
@@ -91,10 +92,20 @@ const redactyl = new Redactyl({
  *
  * @private
  */
-const filterLogData = data =>
-  isPlainObject(data) || data instanceof Error
+const filterLogData = data => {
+  if (data instanceof Error) {
+    /**
+     * serialize Error to plain object to avoid error:
+     *
+     * `TypeError: Cannot convert object to primitive value`
+     */
```

Comment:
This should remain a standard comment so we don't confuse the JSDoc compiler 🙂 
```suggestion
    /*
     * serialize Error to plain object to avoid error:
     *
     * `TypeError: Cannot convert object to primitive value`
     */
```